web/wp-content/themes/dctx/archive.php

```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
+
 	<div class="container clearfix">
 		
 		<div class="main-content">
-			<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
-
 			<?php if (have_posts()) : ?>
```

Comment:
:no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).